### PR TITLE
Minor bugfixes

### PR DESCRIPTION
**Pull request by lmagyar**
PR #130 precedes this PR

# Proposed Changes

See commits:
- fix: jq: error (at <stdin>:1): Cannot iterate over null (null)
- fix: unnecessary "true" or "false" log entry

### CODE REVIEW

**Comment by lmagyar** [comment]
Not stale, waiting for review/merge...

**Comment by lmagyar** [comment]
Not stale, waiting for review/merge...

File: LICENSE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # MIT License
 
-Copyright (c) 2021-2022 Franck Nijhof
+Copyright (c) 2021-2023 Franck Nijhof
```
Comment (frenck) [inline LICENSE.md:3]: Unrelated change for this PR

File: tailscale/rootfs/etc/services.d/tailscaled/post [hunk 2]
```diff
@@ -35,7 +35,7 @@ for address in "${addresses[@]}"; do
 done
 
 # Get configured tags
-tags=$(bashio::config "tags | join(\",\")" "")
+tags=$(bashio::config "tags//[] | join(\",\")" "")
```
Comment (frenck) [inline tailscale/rootfs/etc/services.d/tailscaled/post:38]: Nice 👍

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Rebased the PR, and removed the unrelated documentation changes.

**Review by frenck — APPROVED** [review]
Thanks, @lmagyar 👍 

../Frenck

**Pull request merged**